pnpm-lock.yaml

```diff
@@ -25039,3 +25043,7 @@ packages:
     dependencies:
       eslint: 8.37.0
     dev: true
+
+settings:
+  autoInstallPeers: true
+  excludeLinksFromLockfile: false
```

Comment:
What’s your pnpm version? I’m confused about why this keeps happening.

Reply:
My pnpm version is 8.5.1. i just noticed that we have a volta assertion for pnpm 8.10.5.

Reply:
I updated to pnpm 8.15.4 and re-ran `pnpm install` and it made no changes to the lock file.